File: Leetcode/Java/Easy/3718SmallestMissingMultipleOfK.java
class Solution{
    public int missingMultiple(int[] nums, int k){
        boolean ans_found = false;
        boolean found = false;
        int multiple = 1;
        int ans;
        
        while(ans_found == false){
            ans = multiple * k;
            found = false;
            for(int num : nums){
                if(num == ans){
                    found = true;
                    break;
                }
            }
            if(found == false){
                return ans;
            }
            multiple += 1;
        }
        return 0;
    }
}
//Accepted answer

File: Leetcode/Rust/Easy/3718SmallestMissingMultipleOfK.rs
impl Solution {
    pub fn missing_multiple(nums: Vec<i32>, k: i32) -> i32 {
        let mut found = false;
        let mut ans;
        let mut multiple = 1;
        loop{
            found = false;
            ans = multiple * k;
            for i in 0..nums.len(){
                if(nums[i] == ans){
                    found = true;
                    break;
                }
            }
            if(found == false){
                return ans;
            }
            multiple += 1;
        }

    }
}
//Accepted solution